src/main/resources/application.yml

```diff
@@ -16,7 +16,7 @@ spring:
     enabled: true
 
   datasource:
-    url: ${app.db.url}
```

Comment:
Removing this will break the Camel config - https://github.com/ministryofjustice/keyworker-api/blob/main/src/main/java/uk/gov/justice/digital/hmpps/keyworker/batch/CamelQuartzConfigurationFactory.kt#L27